src/attrib.d

```diff
@@ -740,6 +823,8 @@ public:
                 anonalignsize = 1;
             }
 
+            auto alignment = aligndecl ? aligndecl.getAlignment() : STRUCTALIGN_DEFAULT;
```

Comment:
The best way I can think is, `AnonDeclaration` stores `sc` into its `_scope` instead of `sc.aligndecl`, then use `_scope.alignment()` in here.